packages/multi-select-combo-box/test/visual/common.js

```diff
@@ -4,7 +4,8 @@ registerStyles(
   'vaadin-multi-select-combo-box',
   css`
     /* Hide caret */
-    :host([focused]) ::slotted(input) {
+    :host([focused]:not([has-value])) ::slotted(input),
+    :host([focused][has-value]) ::slotted(input) {
```

Comment:
I had to cover two states explicitly since in the new "pointer focus" tests we have focused input without value, and in the "focused chip" test we have input with value. Maybe could use `:host([focused][focused])` instead.